docs/internals/contributing/writing-documentation.txt

```diff
@@ -341,6 +341,37 @@ documentation:
     Five
     ^^^^
 
+* Add headers for functions, classes, and methods in reference documentation:
+
+  - Each function, class, method, or attribute in reference documentation should
+    have a header using the appropriate heading level based on document hierarchy.
+  - The header should be rendered as fixed width text by wrapping it with
+    double backticks.
+  - Use the exact name of the item as the header text. Headers for functions
+    or methods should have brackets appended, e.g. ``patch_cache_control()``.
+  - Place the header directly above the documentation concerned.
```

Comment:
```suggestion
  - Each function, class, method, or attribute in reference documentation should
    have a header using the appropriate heading level based on document hierarchy.
  - The header should be placed directly above the documentation and rendered
    as fixed width text by wrapping it with double backticks.
  - Use the exact name of the item as the header text. Headers for functions
    or methods should have brackets appended, e.g. ``patch_cache_control()``.
```
Just a small optimisation here - combining two items to make it flow slightly better and be more concise.